recipes/laszip/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 2.8.11)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(${PROJECT_NAME} test_package.c)
+target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
+set_target_properties(${PROJECT_NAME} PROPERTIES LINKER_LANGUAGE CXX)
```

```suggestion
set_target_properties(${PROJECT_NAME} PROPERTIES LINKER_LANGUAGE CXX)
set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11)
```

https://github.com/LASzip/LASzip/blob/3.4.3/CMakeLists.txt#L47

---

unordered_map is not in public headers. It's a C API.

---

fair enough